src/pocketmine/command/defaults/BanIpCommand.php

```diff
@@ -56,7 +56,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 		$value = array_shift($args);
 		$reason = implode(" ", $args);
 
-		if(preg_match("/^([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\.([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\.([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\.([01]?\\d\\d?|2[0-4]\\d|25[0-5])$/", $value)){
+		if(inet_pton($value)){
```

Comment:
this needs an explicit `!== false` check since it can return `string|false`. Not sure why PHPStan didn't fail the build.